docs/content/faq.md

```diff
@@ -22,6 +22,20 @@ See the [remote setup docs](/remote_setup/) for more info.
 
 This has now been documented in its own [remote setup page](/remote_setup/).
 
+### How can I get rid of the "Config file not found" notice?
+
+If you see a notice like 'NOTICE: Config file "rclone.conf" not found', this
+means you have not configured any remotes.
+
+If you need to create a remote, see the [remote setup page](/remote_setup/).
+
+If you are using rclone without remotes, you can create an empty config file
+to get rid of this notice, for example on Linux:
+
+```
+touch ~/.config/rclone/rclone.conf
+```
+
```

Comment:
You can also use `--config ""` which saves making a file.

Reply:
I agree. And if you want the conf file on disk, then [rclone config touch](https://rclone.org/commands/rclone_config_touch/) is a simpler and cross-platform command where you don't have to figure out what the path is.

Reply:
Thanks; I've updated the suggested command to this instead. Seems like the best option.